base_location/models/better_zip.py

```diff
@@ -20,6 +20,8 @@ class BetterZip(models.Model):
     city = fields.Char('City', required=True)
     state_id = fields.Many2one('res.country.state', 'State')
     country_id = fields.Many2one('res.country', 'Country')
+    latitude = fields.Float()
+    longitude = fields.Float()
 
```

Comment:
Note that there is also an `accuracy` in GeoNames that I don't personally care about but would have no problems implementing if someone does.
```
accuracy          : accuracy of lat/lng from 1=estimated to 6=centroid
```